src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +388,116 @@ public InputStream body() throws IOException {
         }
     }
 
+    /**
+     * Fake decorator.
+     * @since 0.16
+     */
+    final class Fake implements RqMultipart {
+
+        /**
+         * Fake multipart request.
+         */
+        private final RqMultipart fake;
+
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         * @todo #252:30min/DEV We should expect instances of Request as dispositions
+         *  Dispositions should be multipart request body holder.
+         *  Ctor. will be called like this.
+         *  {@code
+         *  new RqMultipart.Fake(
+         *  new RqFake("POST", "/upload-a-file"),
+         *  new RqWithHeader(
+         *  new RqFake("", "", "440 Wolfe Rd, CA 94085"),
+         *  "Content-Disposition", "form-data; name=\"address\""
+         *  )
+         *  );
+         *  }
+         */
+        @SuppressWarnings("PMD.UnusedFormalParameter")
```

Comment:
this is very strange, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#constructors